src/js/mixins/AppActionsHandlerMixin.jsx

```diff
@@ -197,9 +197,11 @@ var AppActionsHandlerMixin = {
       if (instancesString != null && instancesString !== "") {
         let instances = parseInt(instancesString, 10);
 
-        if (model.container != null && model.container.volumes != null &&
-          model.instances > instances
-        ) {
+        if (model.container != null &&
```

Comment:
Thanks for noticing. I also realised we should still show the warning if the volume has `.external` defined. Force-pushed some changes.